SpatialGDK/Documentation/content/get-started/tutorial.md

```diff
@@ -150,11 +150,11 @@ To resolve this you need to mark the `CurrentHealth` property for replication, j
 
 Notice that the workflow you just used mirrors that of native Unreal.
 
-Because you’ve changed code in a function you now need to build your project, generate schema and a new snapshot. To do this:
+Because you have changed code in a function, you now need to rebuild your project. Additionally, because you've enabled replication for a variable, you need to generate schema. To do this:
 
 1. Open **ThirdPersonShooter.sln** with Visual Studio.
-1. In the Solution Explorer window, right-click on **ThirdPersonShooter** and select **Build**.
-1. Open **ThirdPersonShooter.uproject** in the Unreal Editor and click `Schema` and then `Snapshot`.
+2. In the Solution Explorer window, right-click on **ThirdPersonShooter** and select **Build**.
```

Comment:
Number indexing is off in this file, they should all be 1s.

Reply:
fixed in https://github.com/spatialos/UnrealGDK/pull/869